pkg/didmethod/peer/did.go

```diff
@@ -27,6 +27,11 @@ const (
 	peerPrefix = "did:peer:"
 )
 
+// nolint:gochecknoglobals
+var (
+	didRegex = regexp.MustCompile(`did:peer:11-([a-fA-F0-9]){64}`)
```

Comment:
why not in one line? 
`var didRegex = regexp.MustCompile(`did:peer:11-([a-fA-F0-9]){64}`)`